.travis.yml

```diff
@@ -13,6 +13,7 @@ addons:
     - myspell-en-za
     - myspell-es
     - myspell-fr
+    - myspell-pt
```

Comment:
Would it be necessary to install `myspell-pt-br` as well, considering that both variants (and others) of Portuguese are used in the same wiki?

Reply:
Aha!  Yes.  But we need to do that in revscoring more than here.  This is just for testing purposes.